CMakeLists.txt

```diff
@@ -134,7 +134,7 @@ if(WITH_GZFILEOP)
     add_definitions(-DWITH_GZFILEOP)
 endif()
 
-if("${CMAKE_C_COMPILER}" MATCHES "icc" OR "${CMAKE_C_COMPILER}" MATCHES "icpc" OR "${CMAKE_C_COMPILER}" MATCHES "icl")
+if("${CMAKE_C_COMPILER_ID}" MATCHES "Intel")
```

Comment:
Should be `if(CMAKE_C_COMPILER_ID MATCHES "Intel")` like the comparisons for GNU further down.

Reply:
I think either works, but if you prefer I will make the change.  Benefits both ways...